### PR TITLE
[Refactoring] - Global typography implementation

### PR DESCRIPTION
**Pull request by UsmanAfzal**
- Abstraction of font sizes from headers for re-usability across components
- Enhancement to alternative sizes
- Compiled list of all component margins to single manageable file
- HTML element includes the body font size to enable increment/decrement

### CODE REVIEW

File: src/stylesheets/components/_hints.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.url {
+	padding-right: $base-spacing-unit;
+	width: 80%;
```
Comment (jakemulley) [inline src/stylesheets/components/_hints.scss]: What if the hint is long - e.g. 25% of space? Might be worth converting to flex.

File: src/stylesheets/settings/_mixins.scss [hunk 2]
```diff
@@ -215,3 +204,73 @@
     }
   }
 }
+
+/*--- Heading Typography Mixins ---*/
+
+@mixin alpha {
```
Comment (jakemulley) [inline src/stylesheets/settings/_mixins.scss]: These mixin names (to me anyway) mean absolutely nothing. Is it worth giving them a more descriptive name?

Comment (UsmanAfzal) [reply src/stylesheets/settings/_mixins.scss]: These are pretty common sources of technical symbols and labels, also used as entities for HTML, CSS content and JS. 

CSS example - `list-style: lower-alpha`
HTML entity example - `&Alpha; &Beta;`

File: src/stylesheets/components/_hints.scss (hunk 1 above)
Comment (UsmanAfzal) [reply src/stylesheets/components/_hints.scss]: Hints were not part of this change just a move around of the existing styles done previously. Won't be looking into as part of this commit.

Comment (jakemulley) [reply src/stylesheets/components/_hints.scss]: 👍

File: src/stylesheets/settings/_mixins.scss (hunk 2 above)
Comment (jakemulley) [reply src/stylesheets/settings/_mixins.scss]: Ah, coolio.

**Pull request merged**